wallet/connect/index.md

```diff
@@ -0,0 +1,55 @@
+
+The following table compares the supported features of each connection option:
+
+| Feature                                                                                                                                                                                                                                                                                            | MetaMask SDK only | Third-party libraries with SDK | Wallet API only |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------:|:------------------------------:|:---------------:|
+| Connect from a web dapp to the MetaMask extension                                                                                                                                                                                                                                                  |         ✓         |               ✓                |        ✓        |
+| Connect from a web dapp, desktop, mobile, and gaming dapps to MetaMask Mobile                                                                                                                                                                                                                      |         ✓         |               ✓                |                 |
+| Connect to MetaMask using the [EIP-6963 provider](../concepts/wallet-interoperability.md)                                                                                                                                                                                                          |         ✓         |               ✓                |        ✓        |
+| Connect to other wallets using the [EIP-6963 provider](../concepts/wallet-interoperability.md)                                                                                                                                                                                                     |                   |               ✓                |        ✓        |
+| Custom RPC methods such as [`connectAndSign`](../how-to/sign-data/connect-and-sign.md), [custom modals](../how-to/display/display-custom-modals.md), [read-only requests](../how-to/make-read-only-requests.md) using Infura API, and [RPC request batching](../how-to/batch-json-rpc-requests.md) |         ✓         |               ✓                |                 |
+
+## MetaMask SDK
+
+The MetaMask SDK ensures a reliable and secure connection to MetaMask Mobile across various platforms. For web dapps, it also enables users to connect via the MetaMask browser extension. The MetaMask SDK can be seamlessly integrated into existing projects without modifying underlying Wallet API calls.
```

Comment:
```suggestion
MetaMask SDK ensures a reliable and secure connection to MetaMask Mobile across various platforms. For web dapps, it also enables users to connect via the MetaMask browser extension. You can seamlessly integrate MetaMask SDK into existing projects without modifying underlying Wallet API calls.
```

- Remove "The" from MetaMask SDK if applicable.
- We try to use active voice whenever it makes sense: https://learn.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice